index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/utility.css">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Micro-Portfolio</title>
+</head>
+<body>
+    <header class="container bg-gray">
+        <div class="flex header">
+            <span class="text-light">John Doe</span>
+            <button class="btn-icon" type="button"> <img src="assets/img/menu.png" alt="menu"></button>
+        </div>
+    </header>
+    <main class=" container home bg-dark">
+        <h1 class="text-primary-1  ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
+        <span class="text-primary-2 ff-roboto mg-1 ">I'm a Software Developer</span>
+        <p class="text-light line-height-2 ff-poppins mg-1">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestitate to contact me.</p>
+        <ul class="flex contact-link mg-1"> 
+           <li> <a href="https://github.com/t-bello7" class="btn-contact"><img src="assets/img/github.png" alt=""> </a>  </li>
```

Comment:
- [ ] Kindly ensure you have a proper background image, this is achieved using a stylesheet adding an image as background [check example here](https://www.w3schools.com/cssref/pr_background-image.asp)